[RHACS] ROX-19788: Adding Image scanning by using roxctl CLI documentation (#66481)
openshift/openshift-docs: operating/examine-images-for-vulnerabilities.adoc

.. In the *Clusters* view, select a cluster to view its details.
.. In the *Health Status* card, ensure that *Scanner* is present and is marked as *Healthy.*

[id="using-image-scanning_{context}"]
==== Using image scanning

> You can scan images stored in a cluster specific {ocp} integrated image registry by using `roxctl` CLI, Jenkins, and API. To do so, specify the appropriate cluster by using the delegated scanning configuration or use the cluster parameter available in `roxctl` CLI, Jenkins, and API.

Reviewer: ```suggestion
You can scan images stored in a cluster specific {ocp} integrated image registry by using `roxctl` CLI, Jenkins, and API. You can specify the appropriate cluster by using the delegated scanning configuration or use the cluster parameter available in `roxctl` CLI, Jenkins, and API.
```

> You can scan images stored in the {ocp} integrated image registry using `roxctl` CLI. To do this, ensure that your systems, including `roxctl` CLI, Jenkins, and API, support delegating scans to specific secured clusters. You can find the project and namespace of the image from it's URI and ensure that you use project-specific credentials for retrieval and scanning.

Reviewer 2: ```suggestion
You can scan images stored in a cluster specific {ocp} integrated image registry using `roxctl` CLI, Jenkins, and API. To do this, specify the appropriate cluster via the delegated scanning configuration or the `cluster` parameter available in the `roxctl` CLI, Jenkins, and API.
```

> For more information on how to scan images by using the `roxctl` CLI, see xref:../cli/image-scanning-by-using-the-roxctl-cli.adoc#image-scanning-by-using-the-roxctl-cli[Image scanning by using the roxctl CLI].

Reviewer: ```suggestion
For more information about how to scan images by using the `roxctl` CLI, see xref:../cli/image-scanning-by-using-the-roxctl-cli.adoc#image-scanning-by-using-the-roxctl-cli[Image scanning by using the roxctl CLI].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i
